aten/src/ATen/native/mps/operations/BinaryOps.mm

```diff
@@ -59,7 +59,7 @@ void binaryOpTensor(const Tensor& self, const Tensor& other, const Scalar& alpha
     output = output_.contiguous();
     needsCopyToOutput = true;
   // else, determine if this is an in-place operation on a view output
-  } else if (output_.is_view() && (self.is_alias_of(output_) || other.is_alias_of(output_))) {
+  } else if (output_.storage_offset() && (self.is_alias_of(output_) || other.is_alias_of(output_))) {
```

Comment:
Why this change is needed ?

Reply:
the change is not needed but the `is_view` is not reliable for check view ops

Reply:
can you create this in a separate PR and add a test case for it ?

Reply:
Yes, I'll separate it out

Reply:
@DenisVieriu97 , I am merging this in as I need to update our master. Can you please create a Test case for the is_view commit.?